components/molecules/HighlightInput/highlight-input-form.tsx

```diff
@@ -528,13 +528,13 @@ const HighlightInputForm = ({ refreshCallback }: HighlightInputFormProps): JSX.E
           </div>
         </div>
       </div>
-      <Dialog
-        onOpenChange={() => {
-          setIsDivFocused(false);
-        }}
-        open={isDivFocused}
-      >
-        <DialogContent className="p-4 w-full md:w-[40rem] xs:w-96 max-h-[80vh] overflow-y-scroll">
+      <Dialog open={isDivFocused}>
+        <DialogContent
+          onEscapeKeyDown={() => {
```

Comment:
I added the <kbd>ESC</kbd> key to make this more accessible on top of just being able to close the modal via the top right close button.